docs/tempo/website/setup/tanka.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Deploy on Kubernetes with Tanka
+weight: 200
+---
+
+# Deploy on Kubernetes with Tanka
+
+Using this deployment guide, you can deploy Tempo to Kubernetes using a Jsonnet library and [Grafana Tanka](https://tanka.dev) to create a development cluster or sand-boxed environment.
+This procedure uses MinIO to provide object storage regardless of the Cloud platform or on-premise storage you use.
+In a production environment, you can use your cloud provider’s object storage service to avoid the operational overhead of running object storage in production.
```

Comment:
```suggestion
In a production environment, you can use your Cloud provider’s object storage service to avoid the operational overhead of running object storage in production.
```
Because you capitalized it in the previous sentence.

Reply:
Good catch. I've corrected the previous one. Cloud shouldn't be upper case because in this case it's not a proper noun.